### ISSUE TITLE
As a public user I would like to recognize at first glance that data products are free of charge

### ISSUE THREAD

**Issue by annegaertner** [dzhw/metadatamanagement#2035]
E. g. during the latest workshop on data ingest in November the symbol of the shopping cart led to confusion regarding possible costs when ordering data sets via the MDM.

I am aware of the fact that the shopping cart represents the common symbol for any kind of orders and of course users need to understand where they have to klick. Furthermore, it cannot be ruled out that costs may arise in the future.

Nevertheless, I currently see a problem with the fact that confusion arises at all. 
Perhaps not even the symbol, but the word "shopping cart" or "Einkaufswagen" needs to be reconsidered. Another solution might be to use prominently placed notes which include that the products are free of charge.

**Comment by rreitmann** [comment]
https://material.io/tools/icons/?style=baseline

**Comment by rreitmann** [comment]
https://materialdesignicons.com/

**Comment by rreitmann** [comment]
The pages above contain material design icons. We are happy for any suggestions to replace the shopping cart with....

**Comment by annegaertner** [comment]
Maybe the following suggestions are worth a discussion:

![basket](https://user-images.githubusercontent.com/31613371/53007241-22035480-3437-11e9-91a3-22f410db2994.jpg)
source: https://materialdesignicons.com/ -> search for "basket"

![basket-fill](https://user-images.githubusercontent.com/31613371/53007360-60990f00-3437-11e9-893d-a43f71175307.jpg)
source: https://materialdesignicons.com/ -> search for "basket-fill"

![package-variant](https://user-images.githubusercontent.com/31613371/53007594-d8ffd000-3437-11e9-86af-c05a5844f0f2.jpg)
source: https://materialdesignicons.com/ -> search for "package-variant"

![sack](https://user-images.githubusercontent.com/31613371/53008028-c3d77100-3438-11e9-95b3-6e3e1cfb940e.jpg)
source:  https://materialdesignicons.com/ -> search for "sack"

These are just first ideas. I am not convinced of any of these so far.

**Comment by rreitmann** [comment]
Great, keep in mind that we need two icons:
1. Showing the state of the shopping cart in the upper right corner
2. Representing the action "add to shopping cart" (button in the lower right corner)

**Comment by annegaertner** [comment]
Thanks for reminding me.
Unfortunately the icons above don't have a second version like we need it.

Is it possible to use selfmade icons so that we (or rather someone with skills) could e. g. pimp one of the icons above?

**Comment by rreitmann** [comment]
Yep, Petra can do it...

**Comment by rreitmann** [comment]
Currently we prefer to give a hint that the data product is free of charge on the study details page as well as in the shopping cart. Price could be added but per access-way. We should postpone this until we have improved the study search and details for the public user...

**Comment by rreitmann** [comment]
Has to be implemented after #2064

**Comment by UteH** [comment]
This issue reoccurred during the usability analysis 😇 

I think we should tackle this issue again soon, be it with a new icon or a banner(?) or an info text.
